geometry_msgs/Point pose        # NOT optional check the ROI that is found to be closest to this pose. could be the robot pose, or pose of a waypoint etc.
---
geometry_msgs/PoseArray output 
